scripts/update_docs.sh

```diff
@@ -4,8 +4,8 @@
 # At present this includes:
```

Comment:
Should we change [scripts/update_docs_local.sh](https://github.com/matrix-org/matrix.org/blob/master/scripts/update_docs_local.sh) as well? Or does that file just seem redundant in general (and we should remove it)?

Reply:
I have actually never used it, and I suspect other things in the scripts are not in sync either. Whether we remove it or get it more in line with the non local one, we can do this in another PR?

Reply:
I had a quick look, it’s indeed out of sync and is basically a(n old) copy with some commented out steps. I reckon a better solution would be to remove the script altogether and make some steps optional in the main one.

Out of scope here.

Reply:
Yep, that was my hunch and thanks for checking. Yes, we can clean it up separately.